File: PlayersNativeiOSApp/PlayersDemo/Views/PlayerDetailView.swift
import SwiftUI

struct PlayerDetailView: View {
    let player: Player
    
    var body: some View {
        VStack {
            Text("Player ID: \(player.player_id)")
            Text("Name: \(player.full_name)")
            Text("Position: \(player.position)")
            Spacer()
        }
        .padding()
    }
}

File: PlayersFlutterApp/players_app/lib/views/player_detail_view.dart
import 'package:flutter/material.dart';
import '../models/player.dart';

class PlayerDetailView extends StatelessWidget {
  final Player player;

  const PlayerDetailView({Key? key, required this.player}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(title: Text(player.fullName)),
      body: Padding(
        padding: const EdgeInsets.all(16.0),
        child: Column(
          crossAxisAlignment: CrossAxisAlignment.start,
          children: [
            Text("Player ID: ${player.playerId}"),
            Text("Position: ${player.position}"),
          ],
        ),
      ),
    );
  }
}
